### PR TITLE
fix: ensure loading spinner shows up on combobox when actions are there

### PR DESCRIPTION
**Pull request by darryltec**
## Motivations

Noticed that when you set the loading to true, the spinner doesn't show up when an action is available. It's because it's being rendered under behind the actions 
image

This fixes it

image

### Added

- 

### Changed

- from fixed to sticky so it stays the the bottom but keeps itself visible when scrolled

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Combobox/components/ComboboxContent/ComboboxContentList/ComboboxContentList.tsx [hunk 1]
```diff
@@ -43,16 +44,15 @@ export function ComboboxContentList(props: ComboboxListProps): JSX.Element {
...
-            </div>
-          )}
+          <div
+            className={classnames([styles.loadingContainer, styles.hasOptions])}
+          >
+            <AnimatedPresence transition="fromBottom">
+              {Boolean(props.loading && optionsExist) && (
+                <Spinner size="small" />
+              )}
+            </AnimatedPresence>
```
Comment (darryltec) [inline packages/components/src/Combobox/components/ComboboxContent/ComboboxContentList/ComboboxContentList.tsx:54]: also added some pizzazz when it shows and hides. It slides from the bottom

**Review by mattander — APPROVED** [review]
LGTM in QA.

Makes sense to me as well. Behaviour is the same since the user has to scroll bottom to trigger anyways. 

The only case covered by `fixed` but not by this is the `onSearch` loading will be hidden. But I don't think that's worth blocking on.

**Review by MichaelParadis — APPROVED** [review]
LGTM!

**Pull request merged**